spec/index.bs

```diff
@@ -6688,8 +6689,14 @@ interface mixin GPURenderEncoderBase {
                      optional GPUSignedOffset32 baseVertex = 0,
                      optional GPUSize32 firstInstance = 0);
 
-    undefined drawIndirect(GPUBuffer indirectBuffer, GPUSize64 indirectOffset);
-    undefined drawIndexedIndirect(GPUBuffer indirectBuffer, GPUSize64 indirectOffset);
+    undefined drawIndirect(GPUBuffer indirectBuffer, GPUSize64 indirectOffset,
```

Comment:
Did you consider adding the extra methods (e.g. `vkCmdDrawIndirectCount`) instead of overloading the existing ones?

Reply:
I considered adding `multiDrawIndirect` and `multiDrawIndexedIndirect`.  No other feature added methods so I was not sure what would be preferred.  I can modify the PR if that is what the committee would prefer.